packager/react-packager/src/Server/index.js

```diff
@@ -502,7 +502,9 @@ class Server {
         data => {
           // Tell clients to cache this for 1 year.
           // This is safe as the asset url contains a hash of the asset.
```

Comment:
Is this comment not true any longer, or is it only referring to the urls of `require(`'d assets? Either way, we should update this comment and explain the intention. Not every app serves to WebViews so people may forget why this is the default.